packages/web-components/fast-components/src/picker/picker-menu-option.styles.ts

```diff
@@ -35,49 +34,49 @@ export const pickerMenuOptionStyles: (
 ) =>
     css`
         :host {
-        display: flex;
-        align-items: center;
-        justify-items: center;
-        font-family: ${bodyFont};
-        border-radius: calc(${controlCornerRadius} * 1px);
-        border: calc(${focusStrokeWidth} * 1px) solid transparent;
-        box-sizing: border-box;
-        color: ${neutralForegroundRest};
-        cursor: pointer;
-        fill: currentcolor;
-        font-size: ${typeRampBaseFontSize};
-        min-height: calc(${heightNumber} * 1px);
-        line-height: ${typeRampBaseLineHeight};
-        margin: 0 calc(${designUnit} * 1px);
-        outline: none;
-        overflow: hidden;
-        padding: 0 calc(${designUnit} * 2.25px);
-        user-select: none;
-        white-space: nowrap;
-    }
+            display: flex;
+            align-items: center;
+            justify-items: center;
+            font-family: ${bodyFont};
+            background: ${neutralFillStealthRest};
+            border-radius: calc(${controlCornerRadius} * 1px);
+            border: calc(${strokeWidth} * 1px) solid transparent;
+            box-sizing: border-box;
+            color: ${neutralForegroundRest};
+            cursor: pointer;
+            fill: currentcolor;
+            font-size: ${typeRampBaseFontSize};
+            min-height: calc(${heightNumber} * 1px);
+            line-height: ${typeRampBaseLineHeight};
+            outline: none;
+            overflow: hidden;
+            padding: 0 calc(((${designUnit} * 3) - 3) * 1px);
+            user-select: none;
+            white-space: nowrap;
+        }
 
-    :host(:${focusVisible}[role="listitem"]) {
-        border-color: ${focusStrokeOuter};
-        background: ${neutralFillRest};
-        color: ${neutralForegroundRest};
-    }
+        :host(:hover) {
+            background: ${neutralFillStealthHover};
+            color: ${neutralForegroundHover};
+        }
 
-    :host(:hover) {
-        background: ${neutralFillHover};
-        color: ${neutralForegroundRest};
-    }
-
-    :host([aria-selected="true"]) {
-        background: ${accentFillRest};
-        color: ${foregroundOnAccentActive};
-    }
+        :host(:active) {
+            background: ${neutralFillStealthActive};
+            color: ${neutralForegroundActive};
+        }
 
+        :host(:not([aria-selected="true"]):hover),
+        :host([aria-selected="true"]) {
+            background: ${neutralFillStealthFocus};
```

Comment:
I think we were using plain neutralFill for selected states,
```suggestion
            background: ${neutralFillRest};
```

Reply:
updated